content/en/docs/kitex/Tutorials/cli-tool/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "CLI Tool"
+linkTitle: "CLI Tool"
+weight: 6
```

Comment:
weight 6 is used by `framework-extern`. Make it 8 which is larger than option and this entry will be placed after option.